tracer/src/Datadog.Trace/Generated/net461/Datadog.Trace.SourceGenerators/AspectsDefinitionsGenerator/AspectsDefinitions.g.cs

```diff
@@ -496,5 +496,297 @@ internal static partial class AspectDefinitions
 "  [AspectMethodReplace(\"System.Text.StringBuilder::Replace(System.Char,System.Char,System.Int32,System.Int32)\",\"\",[0],[False],[None],Default,[])] Replace(System.Text.StringBuilder,System.Char,System.Char,System.Int32,System.Int32)",
 "  [AspectMethodReplace(\"System.Text.StringBuilder::set_Length(System.Int32)\",\"\",[0],[False],[None],Default,[])] SetLength(System.Text.StringBuilder,System.Int32)",
         };
+
+        public static string[] RaspAspects = new string[] {
```

Comment:
I assume you've checked through and made sure this looks like you want! 😃

Reply:
In this particular test it's fine because there are no sinks. :)

Reply:
I mean that the generated code looks correct 😄 (this is the generated code, not the snapshot 🙂)